packages/user/src/views/Login.tsx

```diff
@@ -123,6 +143,7 @@ const Login: React.FC<IProperties> = ({
       className={className}
       data-aria-orientation={orientation}
     >
+      {showReminder()}
```

Comment:
Its not really a reminder, so maybe we can update the name? eg `renderMessage` or `renderRedirectionMessage`